code/lib/initialization.py

```diff
@@ -21,94 +21,94 @@ def initialization():
     # import param and paths
     paths, param = configuration()
 
-    # Read shapefile of scope
```

Comment:
why have you commented this out?

Reply:
I committed this after posting the pull request, it took time to run, and it is not needed to test generate_urbs/evrys_model(). Now, it would be better to wait for me to finish these two functions before merging this branch.